### DISCUSSION TITLE
Umrtia ockovani/neockovani podla vekovych kategorii a celkovy stav zaockovanosti v danej vekovej kategorii

### DISCUSSION THREAD

**Discussion by MiroslavaIvancova** [Institut-Zdravotnych-Analyz/covid19-data#139] [Q&A]
Dobry den,

prosim ktore tabulky si mam pospajat ked potrebujem umrtia v deleni ockovani/neockovani/nezmany stav v cleneni podla vekovej kategorie a este by som tam rada mala celkovy pocet zaockovanych/nezaockovanych/neznamy stav v danej vekovej kategorii. 

Dakujem pekne.

**Comment by MartinHBA** [comment]
umrtia podla ockovania tu nenajdete data

**Reply by MiroslavaIvancova** [reply]
Dakujem pekne za informaciu. Aky je dovod nezverejnovania prosim? Dakujem.

**Reply by MiroslavaIvancova** [reply]
Chapem, ze to nie je v jednej tabulke. Ale predpokladala som, ze to viem dostat pospajanim udajov z viacerych tabuliek, tak len ci sa dobre rozumieme. Vdaka

**Reply by MartinHBA** [reply]
Určite sa to da pospájať cez rodné čísla /s

**Reply by MiroslavaIvancova** [reply]
Dakujem pohladam a ak by bol problem sa ozvem.
